app/models/appointment.rb

```diff
@@ -1,16 +1,16 @@
 class Appointment < ActiveRecord::Base
   validates :name, presence: true
   validates :phone_number, presence: true
-  validates :time, presence: true
+  validates :exactTime, presence: true
```

Comment:
does this work ?   typically we want the column names to be `exact_time` unless there is a REALLY good reason to break this convention.